docs/build.sh

```diff
@@ -8,27 +8,27 @@ set -e
 # We also validate that the tag and version match at this point if needed.
 if [ -n "$CIRCLE_TAG" ]
 then
-  # Check the git tag matches the version number in the VERSION file.
-  VERSION_NUMBER=$(cat VERSION)
-  if [ "v${VERSION_NUMBER}" != "${CIRCLE_TAG}" ]; then
-    echo "Given git tag does not match the VERSION file content:"
-    echo "${CIRCLE_TAG} vs $(cat VERSION)"
-    exit 1
-  fi
-  # Check the version_history.rst contains current release version.
-  grep --fixed-strings "$VERSION_NUMBER" docs/root/intro/version_history.rst \
+    # Check the git tag matches the version number in the VERSION file.
+    VERSION_NUMBER=$(cat VERSION)
+    if [ "v${VERSION_NUMBER}" != "${CIRCLE_TAG}" ]; then
+        echo "Given git tag does not match the VERSION file content:"
+        echo "${CIRCLE_TAG} vs $(cat VERSION)"
+        exit 1
+    fi
+    # Check the version_history.rst contains current release version.
+    grep --fixed-strings "$VERSION_NUMBER" docs/root/intro/version_history.rst \
     || (echo "Git tag not found in version_history.rst" && exit 1)
-
-  # Now that we now there is a match, we can use the tag.
-  export ENVOY_DOCS_VERSION_STRING="tag-$CIRCLE_TAG"
-  export ENVOY_DOCS_RELEASE_LEVEL=tagged
-  export ENVOY_BLOB_SHA="$CIRCLE_TAG"
+    
+    # Now that we now there is a match, we can use the tag.
+    export ENVOY_DOCS_VERSION_STRING="tag-$CIRCLE_TAG"
```

Comment:
Why these changes?

Reply:
Reverted this and the docs/root/api/api.rst changes. Looks like formatting cleanup inserted by my editor. I had made changes in both of these files when the VHDS content was in a separate file.